backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/tax_accounts_details_v.sql

```diff
@@ -50,95 +51,68 @@ SELECT fa.vatcode,
        fa.ad_table_id,
        fa.record_id,
        fa.ad_org_id
-
-FROM (SELECT fa.vatcode                                                                                                                                           AS vatcode,
-             ev.value                                                                                                                                             AS accountno,
-             ev.name                                                                                                                                              AS accountname,
+FROM (SELECT fa.vatcode                        AS vatcode,
+             ev.value                          AS accountno,
+             ev.name                           AS accountname,
              fa.dateacct,
              fa.documentno,
 
-             tax.name                                                                                                                                             AS taxname,
-             tax.rate                                                                                                                                             AS taxrate,
+             tax.name                          AS taxname,
+             tax.rate                          AS taxrate,
 
-             bp.name                                                                                                                                              AS bpName,
+             bp.name                           AS bpName,
 
-             (fa.AmtAcctDr - fa.AmtAcctCr)                                                                                                                        AS taxamt,
-             SIGN(fa.AmtAcctDr - fa.AmtAcctCr)                                                                                                                    AS signTax,
-             COALESCE(i.taxbaseamt, sap_gl.taxbaseamt, gl.taxbaseamt,
+             (fa.AmtAcctDr - fa.AmtAcctCr)     AS taxamt,
+             SIGN(fa.AmtAcctDr - fa.AmtAcctCr) AS signTax,
```

Comment:
TaxSign?

Reply:
but IMHO i don't see a point of having it as a column

where it's needed i just multiply with `* SIGN(TaxAmt)` because
* it's clear
* takes almost the some amount of chars...